libs/directus/directus-next/src/loaders/directusImageLoader/index.ts

```
@@ -0,0 +1,38 @@
+    return src
+  }
+
+  let url: URL
+  try {
+    url = new URL(src)
+  }
+  catch {
+    return src
+  }
+
+  if (!isEmpty(width))
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)